pac4j-core/src/main/java/org/pac4j/core/context/J2EContext.java

```diff
@@ -217,6 +218,7 @@ public String getFullRequestURL() {
     @Override
     public void addResponseCookie(final Cookie cookie) {
         final javax.servlet.http.Cookie c = new javax.servlet.http.Cookie(cookie.getName(), cookie.getValue());
+        c.setVersion(cookie.getVersion());
```

Comment:
The question fits here best: is version 0 the default value? Just to be sure we don't break anything for those who use cookies.